packages/mobile/src/hooks/usePressScaleAnimation.ts

```diff
@@ -6,30 +6,32 @@ export const usePressScaleAnimation = (
   scaleTo = 0.97,
   useNativeDriver = true
 ) => {
-  const scale = useRef(new Animated.Value(1)).current
+  const scaleAnim = useRef(new Animated.Value(1))
```

Comment:
we do this pattern a lot. is that a source of extra renders?

Reply:
yeah i actually dont know for certain, i think probably not, but i tried it here and meant to check again on it and forgot. can remove to be consistent though

Reply:
example in RN docs does .current off the rip, so prob fine: 
<img width="628" alt="image" src="https://user-images.githubusercontent.com/8230000/210037226-b9656483-41de-4b9e-8aa7-28a6f01468e1.png">

the tradeoff is having to put in callback deps which is annoying, but not a huge deal